sim/scenarios/droplist/droplist-error-model.cc

```diff
@@ -0,0 +1,35 @@
+#include <map>
+#include "droplist-error-model.h"
+
+using namespace std;
+
+NS_OBJECT_ENSURE_REGISTERED(DroplistErrorModel);
+ 
+TypeId DroplistErrorModel::GetTypeId(void) {
+  static TypeId tid = TypeId("DroplistErrorModel")
+    .SetParent<ErrorModel>()
+    .AddConstructor<DroplistErrorModel>()
+    ;
+  return tid;
+}
+ 
+DroplistErrorModel::DroplistErrorModel() { }
+
+void DroplistErrorModel::DoReset(void) { }
+ 
+bool DroplistErrorModel::DoCorrupt(Ptr<Packet> p) {
```

Comment:
Should we print a message when we drop a packet, e.g. "Dropping packet x from the client"?

Reply:
I did think about this. I had that earlier and removed it ... it might be good as feedback to the user (I find it useful), so added it now.